recipes/logging_tree/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "logging_tree" %}
+{% set version = "1.9" %}
+{% set sha256 = "fe78b28788c249b515d12a50417e4a96ba095582927b3601a0573886bffc1ff0" %}
```

Comment:
You don't need a jinja variable that is used only once. The recipe is more readable if this goes into line 11.